app/workers/digest_email_generation_worker.rb

```diff
@@ -7,7 +7,7 @@ def perform(subscriber_id:, digest_run_id:)
 
     generate_email_and_subscription_contents
 
-    DeliveryRequestWorker.perform_async_with_priority(email.id, priority: :low)
+    DeliveryRequestWorker.perform_async_for_digest(email.id)
```

Comment:
This already knows about digests so it might be better placed to either specify the queue or use another thing?